dmutils/cloudfoundry.py

```diff
@@ -4,13 +4,14 @@
 
 import json
 import os
+from typing import Dict, List
 
 
 def get_vcap_services() -> dict:
-    return json.loads(os.environ["VCAP_SERVICES"])
+    return json.loads(os.environ["VCAP_SERVICES"])  # type: ignore
```

Comment:
Instead of doing an ignore we can do a typecast with [`cast()`](https://docs.python.org/3/library/typing.html#typing.cast):

```suggestion
    return cast(dict, json.loads(os.environ["VCAP_SERVICES"]))
```

Reply:
Ah, that's a neat thing I was unaware of.

Reply:
Have added casts where I can